File: GTTS/INL.TrainingService.Models/ProjectCodesAtBusinessUnit_Item.cs

namespace INL.TrainingService.Models
{
    public class ProjectCodesAtBusinessUnit_Item : IProjectCodesAtBusinessUnit_Item
    {
        public int ProjectCodeID { get; set; }
        public string Code { get; set; }
        public string Description { get; set; }
        public long BusinessUnitID { get; set; }
        public string Acronym { get; set; }
        public string BusinessUnitName { get; set; }
        public bool BusinessUnitActive { get; set; }
        public bool ProjectCodeBusinessUnitActive { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/project-codes-at-business-unit_item.ts


import { IProjectCodesAtBusinessUnit_Item } from './iproject-codes-at-business-unit_item';

export class ProjectCodesAtBusinessUnit_Item implements IProjectCodesAtBusinessUnit_Item {
  
	public ProjectCodeID: number = 0;
	public Code: string = "";
	public Description: string = "";
	public BusinessUnitID: number = 0;
	public Acronym: string = "";
	public BusinessUnitName: string = "";
	public BusinessUnitActive: boolean = false;
	public ProjectCodeBusinessUnitActive: boolean = false;
  
}






